.github/pr-labeler.yml

```diff
@@ -0,0 +1,4 @@
+enhancement: ["feature/*"]
+bug: ["hotfix/*"]
```

Comment:
Change this to `bugfix/*` for consistency please? We can add a separate hotfix category later if necessary.